script/deploy/local/DeployFromScratchWithChecks.s.sol

```diff
@@ -0,0 +1,374 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "./Deploy_From_Scratch.s.sol";
```

Comment:
note that this file has a ton of duplicated code right now. ideally most of it would be de-duplicated!

Reply:
update: this file is still in bad shape, definitely needs more work

Reply:
file deleted in https://github.com/Layr-Labs/eigenlayer-contracts/pull/794/commits/e8398a1f9b559ca5ae3f5cfd60c2717c210d1e1d
I've been told Zeus development will cover this